python/servo/testing_commands.py

```diff
@@ -57,6 +57,22 @@
 TEST_SUITES_BY_PREFIX = {path: k for k, v in TEST_SUITES.items() if "paths" in v for path in v["paths"]}
 
 
+def taplo(write: bool = False) -> int:
```

Comment:
A person reading this file might not know what taplo is, so I would suggest giving this function a more descriptive name. How about `format_toml_files()`? In addition I would name the argument `check` instead of write and reverse its meaning -- just to match the argument we pass to taplo.